src/cap-description.adoc

```diff
@@ -19,6 +19,19 @@ current draft and expected to be part of privileged architecture 1.13.
 MXLEN without including the tag bit. The value of CLEN is always calculated
 based on MXLEN regardless of the effective XLEN value.
 
+NOTE: We briefly note that the capability encoding described in this section
+could be replaced with an entirely different design without changing how CHERI
+integrates with the RISC-V ISA, and possibly without even changing the ABI. In
```

Comment:
It absolutely is part of the ABI

Reply:
So you'd like to remove this part entirely?
> and possibly without even changing the ABI.

Reply:
Yes

Reply:
Done